packages/sit-onyx/src/components/OnyxStepper/OnyxStepper.vue

```diff
@@ -57,19 +53,54 @@ const value = computed({
   set: (value) => emit("update:modelValue", value),
 });
 
-const handleClick = (direction: "stepUp" | "stepDown") => {
-  if (!inputRef.value) return;
+function countDecimalPlaces(num: number) {
+  const numStr = num.toString();
+  if (numStr.includes(".")) {
+    return numStr.split(".")[1].length;
+  }
+  return 0;
+}
```

Comment:
We can use the standard math functions, this does then works for tens and decimals
```suggestion
function countDecimalPlaces(num: number) {
  return Math.floor(Math.log10(num));
}
```

Reply:
Math.floor(Math.log10(num)) isn't working for every Number. e.g. 1.5 would be 0